galley/pkg/crd/validation/monitoring.go

```diff
@@ -42,6 +42,14 @@ var (
 		Name: "galley_validation_http_error",
 		Help: "Resource validation http serve errors",
 	}, []string{"status"})
+	metricWebhookConfigurationUpdateError = prometheus.NewCounterVec(prometheus.CounterOpts{
+		Name: "galley_validation_config_update_error",
+		Help: "k8s webhook configuration update error",
+	}, []string{"error"})
+	metricWebhookConfigurationUpdates = prometheus.NewCounter(prometheus.CounterOpts{
```

Comment:
The error I reported few times was 'invalid certs' - is it covered by any of the metrics ?

Reply:
Does golang provide a server-side callback for failed tls handshakes? I couldn't find anything when I searched. If not, this type of error needs to be capture by kube-apiserver.